synedrion/src/cggmp21/sigma/fac.rs

```diff
@@ -239,4 +240,29 @@ mod tests {
         let proof = FacProof::<Params>::new(&mut OsRng, &sk, &setup, &aux);
         assert!(proof.verify(pk, &setup, &aux));
     }
+
+    #[test_log::test]
+    fn prove_and_verify_wire_payload() {
```

Comment:
What was the reason for the test? Was there some serialization issue?

Reply:
Moreover, for the sake of consistency, if we have such a test in `fac`, we should have one for all the other ZK proofs as well.

Reply:
No, no particular serialization issues. I just realized that the code involved to deserialize and prepare data for verification is not wholly trivial and felt it'd be good to have a test for that part too.

Maybe I should convert all the tests to actually to the ser/deser? Or would you prefer duplicating the tests like I did here?

Reply:
Technically serialization/deserialization of proofs is already tested by running the protocols, but I see no harm in adding a ser/deser step in the current tests, as you proposed. No need do duplicate everything, I think.